tough/src/datastore.rs

```diff
@@ -65,23 +62,24 @@ impl Datastore {
     }
 
     /// Writes a JSON metadata file in the datastore. This function is thread safe.
```

Comment:
Does "thread safe" still make sense here?

Reply:
It's a bit strange to me since that's not a thing you would generally need to proclaim in rust, but I don't feel a strong urge to remove it. 
tokio requires things to be Sync, too. And in fact tuftool is scheduling multiple tasks that potentially run on different threads.